lib/tmuxinator/cli.rb

```diff
@@ -102,10 +104,12 @@ def start(name, custom_name  = nil)
     end
 
     method_option :attach, :type => :boolean, :aliases => "-a", :desc => "Attach to tmux session after creation."
-    desc "debug [PROJECT] [SESSION_NAME]", COMMANDS[:debug]
+    method_option :name, aliases: "-n",
+      :desc => "Give the session a different name"
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.